web/config/tailwind.config.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
       red: '#ec3750',
       progress: '#ec3750',
     },
+    fontFamily: : {
```

Comment:
It'd be great to also add `mono` and set it to `'SF Mono', 'Roboto Mono', Menlo, Consolas, monospace` as well as adding `header` and set it to `'Shrikhand', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif`

Reply:
Will do!